build/build-versionservice-client.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
+
+set -o errexit
+set -o xtrace
+
+rm -rf versionserviceclient
+
+swagger generate client -f vendor/github.com/Percona-Lab/percona-version-service/api/version.swagger.yaml -c versionserviceclient -m versionserviceclient/models
```

Comment:
Should we integrate it with image build process?

Reply:
No, since we are including this directory to VCS
If you update `version.swagger.yaml` which is versioned with go modules, you will need to regenerate definitions as well